staging/src/k8s.io/apimachinery/pkg/util/httpstream/wsstream/doc.go

```diff
@@ -18,4 +18,4 @@ limitations under the License.
 // The Conn type allows callers to multiplex multiple read/write channels over
 // a single websocket. The Reader type allows an io.Reader to be copied over
 // a websocket channel as binary content.
-package wsstream // import "k8s.io/apiserver/pkg/util/wsstream"
+package wsstream // import "k8s.io/apimachinery/pkg/util/httpstream/wsstream"
```

Comment:
> I agree @aojea that maybe instead of renaming this(which might break consumers), we can deprecate `apiserver/pkg/util/wsstream` and delete it in next release. What do you think?

I don't think it much matters, but ok.

Reply:
upgrading client-go and hitting this renames is 🧑‍🚒